evap/static/scss/_mixins.scss

```diff
@@ -1,7 +1,7 @@
 @import "../bootstrap/scss/mixins";
 
 @mixin bar-shadow($color) {
-    text-shadow: 0 0 4px $color, 0 0 4px $color, 0 0 2px $color;
+    text-shadow: 0 0 2px $color, 0 0 2px $color, 0 0 2px $color, 0 0 2px $color, 0 0 2px $color, 0 0 2px $color;
```

Comment:
Fine with me, I wonder whether we could achieve the same by manipulating the color directly and sticking with one shadow - no idea though :shrug:

Reply:
the problem is that there is not really a high-contrast color on either of the two blues, so we need a shadow to create the necessary contrast, and a smaller number of shadows will not increase the contrast enough.